app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "How to choose your teacher training course"
+subcategory: Postgraduate teacher training
+description: |-
+  Find out how to choose your postgraduate teacher training course based on your eligibility, the qualifications it awards, and the course provider.
+related_content:
+    Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider" 
+    Train to teach if you have a degree : "/train-to-be-a-teacher/if-you-have-a-degree"
+    Your teacher training application : "/how-to-apply-for-teacher-training/teacher-training-application"
+external_content:
+    - content/train-to-be-a-teacher/promos/find-your-course
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-degree
+navigation: 20.25
+navigation_title: How to choose your course
+navigation_description: Find out how to choose your postgraduate teacher training course.
+keywords:
+  - becoming a teacher
+  - teacher training
+  - pgce
+  - qts
+  - Teach First
+  - Postgraduate teaching apprenticeship
+
+---
+
+All postgraduate teacher training courses include time spent in school placements with some theoretical learning. 
+
+The main differences between courses are: 
+
+* what you need to get onto the course 
+* the qualifications the course awards 
+* if the course comes with a salary 
+* the course provider 
+* if it’s full or part-time 
+
```

Comment:
where the course is located?

Reply:
was torn about adding this - obviously it's important to candidates, but it's also maybe important enough that we probably don't need to point it out?

Reply:
Yeah, I would say (and we know from research) candidates are always looking at location so I think it's obvious courses will be in located in different areas. Also that's the first question on Find.